Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -274,6 +300,54 @@ public void onInputBindingClick(final InputBindingSetting item, final int positi
     dialog.show();
   }
 
+  public void onFilePickerDirectoryClick()
+  {
+    FileBrowserHelper.openDirectoryPicker(mView.getActivity());
+  }
+
+  public void onFilePickerFileClick(SettingsItem item)
+  {
+    mFilePicker = (FilePicker) item;
+    mItem = item;
+
+    FileBrowserHelper.openFilePicker(mView.getActivity(), mFilePicker.getRequestType(), false);
+  }
+
+  public static void onFilePickerConfirmation(String file)
+  {
+    NativeLibrary
+            .SetConfig(SettingsFile.FILE_NAME_DOLPHIN + ".ini", mItem.getSection(), mItem.getKey(),
```

Comment:
Could this be made to not hardcode the filename, in case we want to use this for other settings in the future? (I don't understand the code structure here well enough to know how easy it would be, so if it would be too hard, don't worry about it.)

Reply:
Yes, done.